mxd/backend-service/src/test/java/org/eclipse/tractusx/mxd/testfixtures/PostgresRuntime.java

```diff
@@ -23,17 +23,17 @@
 public interface PostgresRuntime {
 
     @RegisterExtension
-    static final BeforeAllCallback CREATE_DATABASE = context -> PostgresqlEndToEndInstance.createDatabase(PostgresqlEndToEndInstance.DB_NAME);
+    BeforeAllCallback CREATE_DATABASE = context -> PostgresqlEndToEndInstance.createDatabase(PostgresqlEndToEndInstance.DB_NAME);
 
     @RegisterExtension
-    public static final EdcRuntimeExtension RUNTIME = new EdcRuntimeExtension(
-            "",
+    EdcRuntimeExtension RUNTIME = new EdcRuntimeExtension(
             "backend",
             Map.of(
                     "edc.datasource.default.url", PostgresqlEndToEndInstance.JDBC_URL_PREFIX + PostgresqlEndToEndInstance.DB_NAME,
                     "edc.datasource.default.user", PostgresqlEndToEndInstance.USER,
                     "edc.datasource.default.password", PostgresqlEndToEndInstance.PASSWORD
-            )
+            ),
+            new String()
```

Comment:
You don't need third argument, varargs can have zero or more arguments.
```
@RegisterExtension
EdcRuntimeExtension RUNTIME = new EdcRuntimeExtension(
        "backend",
        Map.of(....)
);
```

Reply:
It fails if I don't pass an empty string.

Reply:
What error do you get

Reply:
Updated.